src/changelog.ts

```diff
@@ -11,6 +11,11 @@ export interface Change {
 }
 
 export const changelog: ReadonlyArray<Change> = [
+  {
+    date: '2021-04-27',
+    change: 'Swish claims payouts (feature flagged)',
```

Comment:
Maybe IEX don't know what "feature flagged" means 🙈 
```suggestion
    change: 'Swish claims payouts (currently disabled)',
```

Reply:
It's not always gonna be this way though? Maybe we should omit it completely in the changelog until we actually enable it